static/js/demos.js

```diff
@@ -48,7 +53,7 @@ const floating_point_operations_demo_params = {
     ],
 };
 
-const looping_with_tix_demo_params = {
+export const looping_with_tix_demo_params = {
```

Comment:
So is the idea to eventually have these as instances of the Renderer class?

Reply:
I hadn't really thought of that, but it's not a bad idea in ```start_demo``` rather than unpacking the demo_params there, to pass them directly into DemoRender and unpack them in a constructor method that also inits the IBM_704 with the given computer_size. I'm not really sure it's much of a win, though... I'd say let's just leave this structure as is for now.